### DISCUSSION TITLE
Zone C in Accordions

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1704] [Proposals]
### Goal

Avaliar a proposta de adicionar um novo elemento opcional na estrutura do componente Accordion.

### Context

Adicionar uma Zona C para comportar uma predefinição de detalhe, assim como já existe nos componentes Row e Boxed Row.

### Impact

Adicionando essa melhoria, temos mais consistência na experiência da pessoa usuária e na aplicação do componente entre os times.

### Discussion Questions

Seria possível evoluir esse componente em Mística?

### Do you have a visual proposal?

https://www.figma.com/file/vQ11H3lYLBWRyNesHJM1GB/Update-Accordion?type=design&node-id=2705%3A6405&mode=design&t=rgQdkbK7tfje69Un-1

**Comment by yceballost** [comment]
I see no problem to include this!

Just curious, the idea in your example is to show the total of the amount and inside the accordion show the detail of the concepts? like a bill?

**Reply by annefranca** [reply]
Perfeito! Posso abrir um ticket no Jira?

O exemplo foi apenas uma demonstração de um possível caso de uso.

**Reply by yceballost** [reply]
yes please, open a ticket in OBVIVO jira to keep all teams involved aligned

**Reply by annefranca** [reply]
https://jira.tid.es/browse/OBVIVO-2501
